src/renderer/runner.ts

```diff
@@ -20,11 +20,16 @@ export enum ForgeCommands {
   MAKE = 'make',
 }
 
+interface runOptions {
+  runFromAsar: boolean;
+}
+
 interface RunFiddleParams {
   localPath: string | undefined;
   isValidBuild: boolean; // If the localPath is a valid Electron build
   version: string; // The user selected version
   dir: string;
+  runFromAsar: boolean;
```

Comment:
```suggestion
  runFromAsar?: boolean;
```

This is a nit, but personally I'd make this optional.